bin/midi.ml

```diff
@@ -2,6 +2,12 @@ open! Import
 open Result.Syntax
 module Event = Portmidi.Portmidi_event
 
+let bend_pitch ~bend ~timestamp () =
+  let bend_as_integer = int_of_char bend in
```

Comment:
Actually I'd suggest taking the bend parameter as an `int` rather than a `char`.
Because `char` is only a 8 bit value, and you need here to be able to express up to a 14 bits value.
Can you make that change?

Reply:
Ok

Reply:
When I was testing the function in `utop` I noticed that `char_of_int 11709` would not work

Reply:
This is normal as you can only convert to char integers that can be contained in 8 bits. (limit would be 256 when working with unsigned integers.) :-) 

If your bitwise operations are correct, `data1` and `data2` should both be at most 7 bits long, so you shouldn't have any issue!